sbt-mode-comint.el

```diff
@@ -184,14 +184,14 @@ what `sbt:move-marker-before-prompt-filter` did."
 (defun sbt:scala-escape-string (str)
   (mapconcat 'sbt:scala-escape-char str ""))
 
-(defconst sbt:completions-regex "^\\[completions\\] \\(.*\\)
?$")
+(defconst sbt:completions-regex "^\\[completions\\] \\(.*\\)
```

Comment:
is this going to work cross-platform? It seems like we ought to be using a special character class for this instead of an explicit newline